api/v1/swagger.yaml

```diff
@@ -11274,16 +11277,16 @@ definitions:
     description: >
       Gets the config and sample collection metadata from the Refocus server.
     properties:
-      collectorConfigUpdates:
-        $ref: "#/definitions/CollectorConfigUpdatesResponse"
+      collectorConfig:
+        $ref: "#/definitions/CollectorConfigResponse"
       generatersAdded:
         $ref: "#/definitions/GeneratorsAddedResponse"
       generatersUpdated:
         $ref: "#/definitions/GeneratorsUpdatedResponse"
       generatorsRemoved:
         $ref: "#/definitions/GeneratorsRemovedResponse"
 
-  CollectorConfigUpdatesResponse:
+  CollectorConfigResponse:
```

Comment:
The swagger definition of collectorConfig includes many properties which are probably not needed. We can talk about it tomorrow and remove the ones that not required.